docs/source/conf.py

```diff
@@ -27,6 +27,10 @@
 import aiida
 from aiida.manage.configuration import load_documentation_profile
 
+# a reentry scan is required in RTD, to ensure all the API can be imported
+import reentry
```

Comment:
Since this is only needed on RTD (correct me if I'm wrong) should we couch it in the `if ON_RTD` conditional like the other RTD specific code? The `ON_RTD` is defined on line 38

Reply:
Well actually there is no RTD specific code now after `docs_revamp` 😉 
But anyway, my thinking was that if we only run it on RTD, then it won't get tested in CI

Reply:
BTW on a related note: https://github.com/aiidateam/aiida-core/issues/4104#issuecomment-650200606

Reply:
Ah, I didn't realize that got removed in the docs-revamp. Well in that case, we can also remove that environment variable declaration in the conf. But no need to do that here.